### DISCUSSION TITLE
Merging iio-hyprland to this plugin

### DISCUSSION THREAD

**Discussion by JeanSchoeller** [horriblename/hyprgrass#175] [Ideas]
Quick thought:

Should we work on merging [iio-hyprland](https://github.com/JeanSchoeller/iio-hyprland) to this plugin? I feel most touchscreen computers now also kind of support the rotation function as well. iio-hyprland is a simple interface gathering info from iio-sensor-proxy and rotating the screen accordingly using hyprctl.

 What do you guys think?

**Comment by horriblename** [comment]
nope,

1. I'd rather not make it a hyprland plugin if it doesn't need to, since if it crashes it might bring hyprland down as well
2. it's waayyy out of scope for this project

that said, I should add a section in README for "helpful tools on touch screen", since it's not easy to find all of them

**Reply by JeanSchoeller** [reply]
Noted 👍🏽 Didn't take into consideration the plugin aspect of it.
